task/manager.go

```diff
@@ -472,42 +472,37 @@ func (r *Task) pod(addon *crd.Addon, owner *crd.Tackle, secret *core.Secret) (po
 //
 // specification builds a Pod specification.
 func (r *Task) specification(addon *crd.Addon, secret *core.Secret) (specification core.PodSpec) {
+	working := core.Volume{
+		Name: "working",
+		VolumeSource: core.VolumeSource{
+			EmptyDir: &core.EmptyDirVolumeSource{},
+		},
+	}
+	cache := core.Volume{
+		Name: "cache",
+	}
+	if Settings.Cache.RWX {
+		cache.VolumeSource = core.VolumeSource{
+			PersistentVolumeClaim: &core.PersistentVolumeClaimVolumeSource{
+				ClaimName: Settings.Cache.PVC,
+			},
+		}
+	} else {
+		cache.VolumeSource = core.VolumeSource{
+			EmptyDir: &core.EmptyDirVolumeSource{},
+		}
+	}
```

Comment:
When the cache is RWX (supported) then the CACHE PVC is mounted.
Else, an EmptyDir is mounted.